### PR TITLE
new multi-exporter working

### LINKED ISSUE #1125: Complete integration w Leaflet.DistortableImage v0.7.7+ (now v0.8.7) with latest Multi-Image exporter
We had overridden the export code in a previous version, but now that LDI has a proper cloud-based multi-image export system that's quite robust, we should switch to it:

https://github.com/publiclab/mapknitter/blob/f4733a544347916b0367647cf0545ad9b2869fa2/app/assets/javascripts/mapknitter/Map.js#L612-L618

These lines should change, I believe. Referencing https://github.com/publiclab/Leaflet.DistortableImage/issues/316, https://github.com/publiclab/Leaflet.DistortableImage/pull/314, #326

We still don't have docs up at https://github.com/publiclab/Leaflet.DistortableImage#multiple-image-interface-1 but this issue should demonstrate some of it. 

cc @sashadev-sky 👋 🎉

### PR DESCRIPTION
**Pull request by jywarren**
Fixes #1125

🎉 

image

### CODE REVIEW

**Comment by jywarren** [comment]
Going to try to test this out at http://mapknitter-unstable.laboratoriopublico.org/ but not sure we have images there...

**Comment by jywarren** [comment]
Heads up @sashadev-sky ! 😄

**Comment by sashadev-sky** [comment]
woahwoah it is officially externalized!! So exciting

File: app/assets/javascripts/mapknitter/Map.js [hunk 1]
```diff
@@ -661,71 +659,8 @@ MapKnitter.Map = MapKnitter.Class.extend({
...
-            url: "/export",
-            type: 'POST',
-            data: { status_url: statusUrl }
-          }).done(function (data) {
-            console.log('success!! ' + data);
-            updateInterval();
-          });
-        }
-
-        edit.startExport({ handleStatusUrl: addUrlToModel, updater: updateUI, scale: prompt("Choose a scale or use the default (cm per pixel):", 100) });
```
Comment (jywarren) [inline app/assets/javascripts/mapknitter/Map.js]: here's where we had prompted for the scale!

Comment (jywarren) [reply app/assets/javascripts/mapknitter/Map.js]:
```js
edit.startExport({
  handleStatusUrl: addUrlToModel,
  updater: updateUI,
  scale: prompt("Choose a scale or use the default (cm per pixel):", 100)
});
```

**Pull request merged**